### PR TITLE
Fix Groups nav bar issue and update Jasmine tests

### PR DESCRIPTION
**Pull request by kashiish**
These commits fix the id issue for the groups.html page. I've also updated a few Jasmine tests for the groups and group pages that I missed before.

### CODE REVIEW

File: src/main/webapp/groups.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ async function getGroups(group) {
         var groupContainer = document.getElementById("groups-result");
 
         var numSearchResults = document.getElementById("num-groups-results");
-        
+        console.log(numSearchResults);
```
Comment (samfalberg) [inline src/main/webapp/groups.js:55]: Delete?

File: src/main/webapp/group.js [hunk 2]
```diff
@@ -106,8 +106,6 @@ async function getPosts(groupId, document, window) {
         var postsContainer = document.getElementById("posts");
```
Comment (betrevisan) [inline src/main/webapp/group.js:106]: I think the id for this one should be posts-results

File: src/main/webapp/group.js [hunk 3]
```diff
@@ -106,8 +106,6 @@ async function getPosts(groupId, document, window) {
...
 
         var numSearchResults = document.getElementById("num-posts-results");
-
-        postsContainer.appendChild(numSearchResults);
```
Comment (betrevisan) [inline src/main/webapp/group.js:110]: Shouldn't this be kept?

File: src/main/webapp/search-users.js [hunk 4]
```diff
@@ -54,8 +54,6 @@ async function getUsers(name) {
         var userContainer = document.getElementById("users");
 
         var numSearchResults = document.getElementById("num-users-results");
-
-        userContainer.appendChild(numSearchResults);
```
Comment (betrevisan) [inline src/main/webapp/search-users.js:58]: Shouldn't this one also be kept?

File: src/main/webapp/group.js (hunk 3 above)
Comment (kashiish) [reply src/main/webapp/group.js:110]: I added the results label H4 element to the HTML directly, so we can just get it by calling getElementById.

File: src/main/webapp/search-users.js (hunk 4 above)
Comment (kashiish) [reply src/main/webapp/search-users.js:58]: Same thing here.

**Pull request merged**